src/android/java/com/pspdfkit/cordova/Utilities.java

```diff
@@ -40,4 +44,55 @@ public static String convertJsonNullToJavaNull(@Nullable String value) {
     if (value == null || value.equals("null")) return null;
     return value;
   }
+
+  /**
+   * Converts given string for annotation type into a corresponding {@link EnumSet<AnnotationType>}
+   * @param type string for annotation type to convert
+   * @return corresponding {@link EnumSet<AnnotationType>}
+   */
+  public static EnumSet<AnnotationType> getTypeFromString(@Nullable String type) {
```

Comment:
Moved in here for now. In case if `Utilities.java` will bloat could be split to a separate annotation specific utility class.